TrkDiag/fcl/prolog.fcl

```diff
@@ -8,30 +8,23 @@ BD : {
   StrawDigiMCCollection : "compressDigiMCs"
   ComboHitCollection : "makePH"
   StrawHitFlagCollection : "FlagBkgHits:ComboHits"
-  BackgroundHitClusterCollection:"FlagBkgHits"
-  UseFlagCollection : true
-  TimeOffsets : {
-    inputs : [ @sequence::DigiCompression.TimeMaps ]
-  }
+  BkgQualCollection:"FlagBkgHits"
```

Comment:
Just out of curiosity, what is replacing the TimeOffset? Is that done without any config now?

Reply:
TimeOffset is no longer neede.  Proton bunch time profile is applied in the beam campaign, muon lifetime in the Primary and pileup.  References to TimeOffset should be removed opportunistically.